src/JMS/Serializer/Metadata/ClassMetadata.php

```diff
@@ -34,6 +34,7 @@ class ClassMetadata extends MergeableClassMetadata
     const ACCESSOR_ORDER_UNDEFINED = 'undefined';
     const ACCESSOR_ORDER_ALPHABETICAL = 'alphabetical';
     const ACCESSOR_ORDER_CUSTOM = 'custom';
+    const COUNT_PARAMS = 16;
```

Comment:
In my opinion caches are not part of backwards compatibility. One can expect the users to clean caches.